RadixWallet/WalletApp.swift

```diff
@@ -6,6 +6,15 @@ import SwiftUI
 struct WalletApp: SwiftUI.App {
 	@UIApplicationDelegateAdaptor var delegate: AppDelegate
 
+	init() {
+		#if DEBUG
+		// MUST NOT be called twice (panics...)
+		// MUST NOT be enabled in PROD (not ensured that secrets are omitted)
+		// This is VERY VERY verbose, so mosly useful for debugging.
+		enableLoggingFromRust()
```

Comment:
VERY verbose, we might wanna disable it by default, but lets try this for now... see here, how verbose: https://github.com/radixdlt/babylon-wallet-ios/pull/1113